plugin/endwise.vim

```diff
@@ -63,9 +63,9 @@ augroup endwise " {{{1
         \ let b:endwise_pattern = '^\s*\(d\zsef\zeine\|\zsif\zen\=\(def\|eq\)\)\>' |
         \ let b:endwise_syngroups = 'makePreCondit,makeDefine'
   autocmd FileType verilog,systemverilog
-        \ let b:endwise_addition = 'end&' |
+        \ let b:endwise_addition = '\=submatch(0)=="begin" ? "end" : "end" . submatch(0)' |
```

Comment:
Please make this `==#` for clarity.

Reply:
Ok, I copied it from [line 36](https://github.com/tpope/vim-endwise/blob/master/plugin/endwise.vim#L36), shouldn't it be `==#` there as well?